src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/ValuesSources/WeaklyCachedValueSource.cs

```diff
@@ -31,12 +31,13 @@ public WeaklyCachedValueSource(ValueSource<T> source)
 
         private SemaphoreSlim Gate => LazyInitialization.EnsureInitialized(ref _lazyGate, SemaphoreSlimFactory.Instance);
 
-        public override bool TryGetValue([MaybeNullWhen(false)]out T value)
+#pragma warning disable CS8610 // Nullability of reference types in type of parameter doesn't match overridden member. (The compiler incorrectly identifies this as a change.)
```

Comment:
Is there a tracking bug to link to here? Your "incorrectly" implies we have a bug here.

Reply:
#41884

Reply:
Want to put the comment in the bug, so we can figure out which #pragmas to disable later via grep?